generators/app/templates/src/index.js

```diff
@@ -0,0 +1,46 @@
+export default {
+    // Multiple browsers support
+    isMultiBrowser: <%= isMultiBrowser %>,
+
+
+    // Required - must be implemented
+    // Browser control
+    async openBrowser (/* id, pageUrl<%if (isMultiBrowser) {%>, browserName<% } %> */) {
+        throw new Error('Not implemented!');
+    },
+
+    async closeBrowser (/* id */) {
+        throw new Error('Not implemented!');
+    },
+
+
+    // Optional - implement methods you need, remove other methods
+    // Initialization
+    async init () {
+        return;
+    },
+
+    async dispose () {
+        return;
+    },
+
+    <% if (isMultiBrowser) { %>
+    // Browser names handling
+    async getBrowserList () {
+        throw new Error('Not implemented!');
+    },
+
+    async isValidBrowserName (/* browserName */) {
+        return true;
+    },
+    <% } %>
+
+    // Extra methods
+    async resizeWindow (/* id, width, height, currentWidth, currentHeight */) {
+        this.reportWarning('The screenshot functionality is not supported by the "<%= providerName %>" browser provider.');
```

Comment:
For the `resizeWindow` method, the warning text for the takeScreenshot method is used. Move this text to `takeScreenshot`.